scrapy_zyte_api/providers.py

```diff
@@ -146,6 +166,32 @@ async def __call__(  # noqa: C901
             results.append(response)
             self.update_cache(request, {BrowserResponse: response})
 
+        if HttpOrBrowserResponse in to_provide:
+            if extract_from == "browserHtml":
```

Comment:
I'm not sure I understand this part. Why is it tied to extract_from? I was thinking the logic is this: if you have BrowserHtml in the output, you use it to create HttpOrBrowserResponse. If not, you use HttpResponse. It seems at this point extract_from shouldn't matter.

Reply:
Good point. Updated in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/161/commits/3bc8b1462d58b5039f228e345173af0a05f38113